v4/export/dump_test.go

```diff
@@ -49,19 +54,24 @@ func (s *testSQLSuite) TestDumpBlock(c *C) {
 	taskChan := make(chan Task, 1)
 	taskChan <- &TaskDatabaseMeta{}
 	d.conf.Tables = DatabaseTables{}.AppendTable(database, nil)
-	c.Assert(errors.ErrorEqual(d.dumpDatabases(writerCtx, conn, taskChan), context.Canceled), IsTrue)
-	c.Assert(errors.ErrorEqual(wg.Wait(), writerErr), IsTrue)
+	require.True(t, errors.ErrorEqual(d.dumpDatabases(writerCtx, conn, taskChan), context.Canceled))
+	require.True(t, errors.ErrorEqual(wg.Wait(), writerErr))
```

Comment:
can we use `require.ErrorIs` for these two

Reply:
Let me have a try.

Reply:
Updated.